data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,293 @@
+			"With all the different species we have to [keep track of, document], it is easy to get [mixed up, confused]. A spiny-haired T'kai may be upon closer [inspection, study] a short-necked Kat'tchi."
+		word
+			`"`
+
+news "Wanderer soldier"
+	to show
+		has "Wanderers Invaded 0: accepted"
+	location
+		government "Wanderer"
+	name
+		word
+			"Wanderer soldier"
+			"Soldier"
+			"Wanderer conscript"
+			"Conscript"
+	message
+		word
+			`"`
+		word
+			"I wish we did not have to [fight, do battle] to survive. It is a [savage, inelegant] way of solving conflict."
+			"I joined our new [defense fleet, navy] because I wanted to protect lifeforms. Instead I am [battling, killing] them. It brings me great [sadness, sorrow] to see such [destruction, waste]."
+			"I volunteered for service in a warship, and many [ask, ponder] why. For every Hai I [shoot down, kill] I save [many, multiple] civilians."
+			"Just a few days ago I was asked to form a new [military, army] unit, even though I was given almost no [training, experience] in war. These are [desperate, hopeless] times indeed."
+			"Our [commander, unit leader] used to be a zoologist. His only [combat, fighting] experience was defending himself against a rabid Chi'kta on an [expedition, study]. They couldn't find anyone [better, more suitable], so he got 'promoted'."
+			"One of my crewmates is [scared, frightened] of the sound of guns firing. How he got conscripted is [anyone's guess, a mystery]."
+			"Many of our [commanders, leaders] have no idea how a military [works, operates]. They often have to consult history [books, records] so that they can [learn, study] the [workings, activities] of a proper army."
+		word
+			`"`
+
+news "Wanderer engineer"
+	to show
+		has "Wanderers Solifuge Recon 2: offered"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Wanderer engineer"
+			"Engineer"
+	message
+		word
+			`"`
+		word
+			"The warships we have begun to [manufacture, build] are from a more [brutish, dark] time in our history. I hope that we are not headed towards such times [again, once more]."
+			"Despite their [savage, regrettable] purpose, the warships we have begun to [make, fashion] are impressive from a purely technological [perspective, view]."
+			"I regret that I must [recreate, unearth] technologies of [war, battle], but it is necessary for our [survival, existence]."
+			"I would like to [study, understand] human technology and its [capabilities, abilities], but I am not able to. Now that we must build [warships, battleships], all my time has been taken up by their [production, creation]."
+			"I create weapons of [destruction, annihilation] so that we may survive and go on to [create, rebirth]."
+		word
+			`"`
+
+news "Wanderers and the Eye"
+	to show
+		has "Wanderers Ap'arak 3: completed"
```

Comment:
Same as before, a more appropriate condition might be the following:
```suggestion
		has "event: wanderers: the eye opens"
```
(Both the suggestions are to use synonymous conditions in terms of when they appear, I'd just prefer using the condition that is the most direct trigger for these messages displaying instead of using the mission that triggers the condition/event that these are based on.)